ts/components/wallet/card/SectionCardComponent.tsx

```diff
@@ -194,56 +195,50 @@ const SectionCardComponent: React.FunctionComponent<Props> = (props: Props) => {
 
   return (
     <>
-      {Platform.OS === "android" && <View style={styles.shadowBox} />}
-      <View style={[styles.rotateCard]}>
-        <TouchableDefaultOpacity
-          onPress={onPress}
-          accessible={true}
-          accessibilityRole={"button"}
+      {Platform.OS === "android" && (
+        <View
+          accessible={false}
+          accessibilityElementsHidden={true}
+          importantForAccessibility={"no-hide-descendants"}
+          style={styles.shadowBox}
+        />
+      )}
+      <TouchableDefaultOpacity
+        style={styles.rotateCard}
+        onPress={onPress}
+        accessible={true}
+        accessibilityRole={"button"}
+      >
+        <View
+          style={[styles.card, styles.flatBottom, cardStyle || styles.cardGrey]}
         >
           <View
-            style={[
-              styles.card,
-              styles.flatBottom,
-              cardStyle || styles.cardGrey
-            ]}
+            style={[styles.cardInner]}
+            accessibilityLabel={props.accessibilityLabel}
+            accessibilityHint={I18n.t("wallet.accessibility.sectionCardHint")}
```

Comment:
In this case shouldn't be something different, like "Aggiorna la lista dei bonus e sconti"? We have:
"Aggiorna bonus e sconti, pulsante, Aggiorna la lista dei metodi di pagamento".

Reply:
my bad, forgot to spread the accessibility hint prop as for the label commit c34a4f8 implements it